test/quicksort.jl

```diff
@@ -0,0 +1,126 @@
+"""
+Unit tests for quicksort.
+Usage
+```
+(@v1.5) pkg> activate "C:\\Users\\Alex\\Documents\\Github\\CUDA.jl"
+ Activating environment at `C:\\Users\\Alex\\Documents\\Github\\CUDA.jl\\Project.toml`
+
+julia> include("C:\\Users\\Alex\\Documents\\GitHub\\CUDA.jl\\test\\quicksort.jl")
+[ Info: Precompiling CUDA [052768ef-5323-5732-b1bb-66c8b64840ba]
+Test Passed
+
+julia>
+```
+"""
+
+using DataStructures, Random, Test
+
+#using CUDA
+include("../src/sorting/quicksort.jl")
```

Comment:
CUDA.jl should just include this functionality unconditionally. I assume you left it out for easier development? With Revise that shouldn't be necessary though.